### ISSUE TITLE
4/15/2020 + Covid-19 Data Analysis + 5

### ISSUE THREAD

**Issue by anazariz** [lzim/teampsd#1241]
**1. Add description:** 

Covid-19 - data integration reseach

<< Paste screenshots here>>
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by lzim** [comment]
@anazariz @anthonycpichardo @staceypark 

I believe we may have some COVID-19 card proliferation...

- Can we clarify what is what in terms of these cards - **My preference is to _title_ the cards the way @staceypark did according to phases of the _MTL Covid-19 Release_: DEFINE, DESIGN, DEVELOP, TEST, PROD)**
a) key deliverables @staceypark created for the **covid19 release** (aka NIDA & HSRD grants)

b) these cards @anazariz created a while back - **My preference is to standardize** the requirements and testing, etc. according to #921 **master feature card** standardization

c) **mtl_code** secure issues for code review... 
- Also, let's get accurate due dates - this is today! ⌛ 

Cross-ref:
#1309 
#1308
#1307 
#1306 
#1282 
#1242

**Comment by anthonycpichardo** [comment]
Come to 09/02 quant research meeting with initial hypothesis after review of the Margaret Cho work.
